stack.yaml

```diff
@@ -26,7 +26,8 @@ extra-deps:
 - http-download-0.2.0.0@rev:0
 - github: snoyberg/filelock
   commit: 97e83ecc133cd60a99df8e1fa5a3c2739ad007dc
-- pantry-0.3.0.0@rev:0
+- github: commercialhaskell/pantry
+  commit: 7ac9f9364bf8491567f3d00d948bbf9c84cca5b8
```

Comment:
What's the motivation for not using a proper version from Hackage?

Reply:
I wanted to test this PR and get review before committing to a release. With your review, I'll go ahead and release + update this PR.